src/btree/bt_compact.c

```diff
@@ -173,23 +173,24 @@ __compact_page(WT_SESSION_IMPL *session, WT_REF *ref, bool *skipp)
     WT_REF_LOCK(session, ref, &previous_state);
 
     /*
-     * Skip deleted pages but consider them progress (the on-disk block is discarded by the next
-     * checkpoint).
-     */
-    if (previous_state == WT_REF_DELETED)
-        *skipp = false;
+     * Skip obsolete deleted pages (the on-disk blocks are already discarded) */
```

Comment:
The comment formatting doesn't seem right here. It should be either single line, or have the terminator on it's own line (why didn't some automatic tool pick this up?)

Reply:
My mistake